content/guides/multiaccountorg.md

```diff
@@ -41,7 +41,11 @@ The Multi-Account Organizations feature must be enabled by support. If this is a
 1. After the feature has been enabled, visit the [New Account Page](https://app.datadoghq.com/account/new_org).
 2. Enter the name of the organization you wish to create and click the **Create** button.
 
-    {{< img src="guides/multiaccountorg/guides-multacct-createorg.png" alt="Create Org" >}}
+<div class="alert alert-warning">
+Please be aware that your organization name should be <b>less than 32 characters</b>.
```

Comment:
should be, or must be? Also, is it 32 characters inclusive, or is 31 the max?

Also, I think we're going a little overboard with the warning styling lately. How about just a short statement in bold at the end of step 2?

"2. Enter the name of the organization you wish to create and click the **Create** button. **The organization name cannot exceed 32 characters.**"